File: Examples/CSharp/ManageBarcodeImages/RotateBarcode.cs
using Aspose.BarCode.Generation;

/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.BarCode for .NET API reference 
when the project is build. Please check https://docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.BarCode for .NET API from http://www.aspose.com/downloads, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using http://www.aspose.com/community/forums/default.aspx
*/

namespace Aspose.BarCode.Examples.CSharp.ManageBarCodeImages
{
    class RotateBarcode
    {
        public static void Run()
        {
            // ExStart:RotateBarcode       
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_ManageBarCodesImages();

            // Instantiate barcode object and set CodeText, Barcode Symbology and  hide code text
            BarcodeGenerator  generator = new BarcodeGenerator (EncodeTypes.QR);
            generator.CodeText = "1234567890";
            generator.Parameters.Barcode.CodeTextParameters.Location = CodeLocation.None;
            generator.Parameters.RotationAngle = 90;
            generator.Save(dataDir + "QR-rotate_out.jpeg", BarCodeImageFormat.Jpeg);
            // ExEnd:RotateBarcode       
        }
    }
}

File: Examples/VisualBasic/ManageBarcodeImages/RotateBarcode.vb
Imports System.IO
Imports Aspose.BarCode
Imports Aspose.BarCode.Generation

Namespace Aspose.BarCode.Examples.VisualBasic.ManageBarCodeImages
    Class RotateBarcode
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_ManageBarCodesImages()

            ' Instantiate barcode object and set CodeText, Barcode Symbology and  hide code text
            Dim barCodeBuilder As New BarCodeBuilder("1234567890", EncodeTypes.QR) With { _
                .CodeLocation = CodeLocation.None, _
                .RotationAngleF = 90 _
            }
            BarCodeBuilder.Save(dataDir & Convert.ToString("QR-rotate_out.jpeg"), BarCodeImageFormat.Jpeg)
        End Sub
    End Class
End Namespace